<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Credit</class>
 <widget class="QWidget" name="Credit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #202020;</string>
  </property>
  <widget class="QDoubleSpinBox" name="creditAmount">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>170</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QDoubleSpinBox {
background-color: #323232;
color: #ffffff;
font: 14pt &quot;Sans Serif Collection&quot;;
border-radius: 5%;
margin: 2;
}

QDoubleSpinBox:hover {
background-color: #3B3B3B;
}

QDoubleSpinBox::up-button { width: 20px;
border:0;
}

QDoubleSpinBox::down-button { width: 20px;
border: 0;
}</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="interestRate">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>50</y>
     <width>170</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QDoubleSpinBox {
background-color: #323232;
color: #ffffff;
font: 14pt &quot;Sans Serif Collection&quot;;
border-radius: 5%;
margin: 2;
}

QDoubleSpinBox:hover {
background-color: #3B3B3B;
}

QDoubleSpinBox::up-button { width: 20px;
border:0;
}

QDoubleSpinBox::down-button { width: 20px;
border: 0;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: white;
font: 14pt &quot;Sans Serif Collection&quot;;
}</string>
   </property>
   <property name="text">
    <string>Credit amount</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>243</x>
     <y>20</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: white;
font: 14pt &quot;Sans Serif Collection&quot;;
}</string>
   </property>
   <property name="text">
    <string>Month Term</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>425</x>
     <y>20</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: white;
font: 14pt &quot;Sans Serif Collection&quot;;
}</string>
   </property>
   <property name="text">
    <string>Interest rate</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="annuity">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>170</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton {
padding-left: 10px;
background-color: #323232;
color: #ffffff;
font: 14pt &quot;Sans Serif Collection&quot;;
border-radius: 5%;
margin: 2;
}

QRadioButton:hover {
background-color: #3B3B3B;
}

QRadioButton:pressed {
background-color: #282828;
color: #cecece;
}

QRadioButton::indicator {
width: 14px;
height: 14px;
border-radius: 9px;
border: 2px solid  #ffffff;
}

QRadioButton::indicator:checked {
width: 10px;
height: 10px;
border: 4px solid  #9690c2;
border-radius: 9px;
}</string>
   </property>
   <property name="text">
    <string>Annuity</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="differentiated">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>100</y>
     <width>170</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton {
padding-left: 10px;
background-color: #323232;
color: #ffffff;
font: 14pt &quot;Sans Serif Collection&quot;;
border-radius: 5%;
margin: 2;
}

QRadioButton:hover {
background-color: #3B3B3B;
}

QRadioButton:pressed {
background-color: #282828;
color: #cecece;
}

QRadioButton::indicator {
width: 14px;
height: 14px;
border-radius: 9px;
border: 2px solid  #ffffff;
}

QRadioButton::indicator:checked {
width: 10px;
height: 10px;
border: 4px solid  #9690c2;
border-radius: 9px;
}</string>
   </property>
   <property name="text">
    <string>Differentiated</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonCalculate">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>100</y>
     <width>170</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: #b5adeb;
color: #2e2c3c;
font: 14pt &quot;Sans Serif Collection&quot;;
border-radius: 5%;
margin: 2;
}

QPushButton:hover {
background-color: #9690c2;
}

QPushButton:pressed {
background-color: #9690c2;
color: #605c7c;
}</string>
   </property>
   <property name="text">
    <string>Calculate</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="monthTerm">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>50</y>
     <width>170</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox {
background-color: #323232;
color: #ffffff;
font: 14pt &quot;Sans Serif Collection&quot;;
border-radius: 5%;
margin: 2;
}

QSpinBox:hover {
background-color: #3B3B3B;
}

QSpinBox::up-button { width: 20px;
border:0;
}

QSpinBox::down-button { width: 20px;
border: 0;
}</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="output">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>541</width>
     <height>271</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextBrowser {background-color: #323232;
color: #ffffff;
font: 24pt &quot;Sans Serif Collection&quot;;
border-radius: 5%;
margin: 2;
}

QScrollBar:vertical {
width: 0px;
}</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
